src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -59,6 +63,7 @@ const ReactErrorUtils = {
   /**
    * Same as invokeGuardedCallback, but instead of returning an error, it stores
    * it in a global so it can be rethrown by `rethrowCaughtError` later.
+   * TODO: See if _caughtError and _rethrowError can be unified.
```

Comment:
I probably misunderstood, thanks for clarifying. I thought the flag only existed because null/undefined is a valid error.